### ISSUE TITLE
Hero Animation Support?

### ISSUE THREAD

**Issue by MarsGoatz** [jpnurmi/cupertino_list_tile#2]
I have a use case where I need Hero animation on the Cupertino List Tile, everything seems to work fine except the divider which isn't smooth and jumps from initial position to the final position. 

Do you know any workaround for this? or is this something you would be willing to support in later releases?

Thank you!

------------------------

Here is a quick example:

```
import 'package:cupertino_list_tile/cupertino_list_tile.dart';
import 'package:flutter/cupertino.dart';
import 'package:flutter/material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.orange,
      ),
      home: MyHomePage(),
    );
  }
}

class MyHomePage extends StatefulWidget {
  @override
  _MyHomePageState createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
        backgroundColor: Colors.black,
        appBar: AppBar(
          title: Text('Cupertino List Tile Hero Test'),
        ),
        body: Container(
          height: double.infinity,
          width: double.infinity,
          child: Center(
            child: Hero(
              tag: Key('test'),
              child: CupertinoListTile(
                leading: Icon(
                  Icons.settings,
                  color: Colors.white,
                  size: 18,
                ),
                title: Text(
                  'Settings',
                  style: TextStyle(color: Colors.white),
                ),
                subtitle: Text(
                  'Change functionality here',
                  style: TextStyle(color: Colors.white),
                ),
                onTap: () {
                  Navigator.push(context,
                      CupertinoPageRoute(builder: (_) => MyHomePage2()));
                },
              ),
            ),
          ),
        ));
  }
}

class MyHomePage2 extends StatefulWidget {
  @override
  _MyHomePage2State createState() => _MyHomePage2State();
}

class _MyHomePage2State extends State<MyHomePage2> {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
        backgroundColor: Colors.black,
        appBar: AppBar(
          title: Text('Cupertino List Tile Hero Test'),
        ),
        body: Container(
          child: Hero(
            tag: Key('test'),
            child: CupertinoListTile(
              leading: Icon(
                Icons.settings,
                color: Colors.white,
                size: 18,
              ),
              title: Text(
                'Settings',
                style: TextStyle(color: Colors.white),
              ),
              subtitle: Text(
                'Change functionality here',
                style: TextStyle(color: Colors.white),
              ),
              onTap: () {
                Navigator.pop(context);
              },
            ),
          ),
        ));
  }
}

```

**Comment by jpnurmi** [comment]
Thanks for taking the time to provide an example to reproduce the issue. I don't have a workaround to provide right now, but it certainly would be nice to get this fixed.

**Comment by jpnurmi** [comment]
![hero](https://user-images.githubusercontent.com/140617/90322253-892d9680-df52-11ea-8285-940b19b1d856.gif)

**Comment by jpnurmi** [comment]
Fix published in version 0.1.2. :)

**Comment by MarsGoatz** [comment]
Whoa! Thank you for such a quick fix!
